Jenkinsfile

```diff
@@ -27,7 +28,7 @@ pipeline {
 					nvm(getNodejsVersion()) {
 						sh '''
 						rm -rf lisk-service/ # linting will fail otherwise
-						npm run lint
+						yarn run lint
```

Comment:
yarn doesn't need to have the `run` keyword before it can run any script so `yarn lint` works fine. Other instances can be updated accordingly